libssh-rs/src/sftp.rs

```diff
@@ -218,12 +218,12 @@ impl Sftp {
     pub fn open(
         &self,
         filename: &str,
-        accesstype: c_int,
+        accesstype: OpenFlags,
```

Comment:
since this changes the API in a backwards incompatible way, please also bump the version in the `Cargo.toml` to `0.3.0`

Reply:
I bumped both the library and the -sys library versions since it looked like their versions are updated in lockstep, let me know if I should only update libssh-rs.

Reply:
the -sys version doesn't need a bump; it's interface didn't change as part of this PR

Reply:
Changed the -sys crate version back to 0.2.2